package exception;

import java.util.Random;
import java.util.Scanner;

public class ExceptionTest5 {
	public static void main(String[] args) {
//		int n = (int) (Math.random() * 99) + 1;
		Random random = new Random();
		int n = random.nextInt(99) + 1;
		
		try (Scanner sc = new Scanner(System.in)) {
			while(true) {
				System.out.println("1부터 99 사이의 정수를 입력하세요 > ");
				if (sc.hasNextInt()) {
					System.out.println("값이 잘못 입력되었습니다.");
				}
			}
		}
	}
}
